error.go

```diff
@@ -221,3 +221,29 @@ var tooManyResultsError = &microerror.Error{
 func IsTooManyResults(err error) bool {
 	return microerror.Cause(err) == tooManyResultsError
 }
+
+var (
+	yamlConvertingFailedRegexp = regexp.MustCompile(`error converting YAML to JSON:`)
```

Comment:
```suggestion
	yamlConversionFailedRegexp = regexp.MustCompile(`error converting YAML to JSON:`)
```

I think this naming is clearer. Can you move regex to the top of the file like in the errors library.

Reply:
I think there is no reason for a regular expression. It looks like you only need to check for a string match. https://golang.org/pkg/strings/#Contains is your friend.